aws/main.mk

```diff
@@ -15,8 +15,8 @@ AWS_ACCOUNT ?= $(shell [ -f ~/.aws/credentials ] && $(AWS) sts get-caller-identi
 AWS_DEV_ENV_NAME ?= $(shell [ -f ~/.aws/credentials ] && $(AWS) iam list-user-tags --user-name $(AWS_USER) | ( $(JQ) -e -r '.Tags[] | select(.Key == "devEnvironmentName").Value') || echo "$(ENV) (User env is not configured)")
 
 # $(AWS_ARGS) definition see in .infra/icmk/aws/localstack.mk
-AWS ?= $(DOCKER) run \
-	-v $(HOME)/.aws/:/root/.aws \
+AWS ?= $(DOCKER) run --user "$(CURRENT_USER_ID):$(CURRENT_USERGROUP_ID)" \
+	-v $(HOME)/.aws/:/.aws \
```

Comment:
Are we mounting `.aws` to the root directory? Does this work with aws cli and standard SDK (for third-party tools like Terraform)?

Reply:
yes, it works